src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -762,10 +762,15 @@ def list_revision(cmd,
     label = prep_label_filter_for_url_encoding(label)
 
     try:
+        query_fields = []
+        if fields:
+            for field in fields:
+                query_fields.append(field.name.lower())
+
         revisions_iterable = azconfig_client.list_revisions(key_filter=key,
                                                             label_filter=label,
                                                             accept_datetime=datetime,
-                                                            fields=fields)
+                                                            fields=query_fields)
```

Comment:
>query_fields [](http://example.com/codeflow?start=67&length=12)

Based on the [sdk comments](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py#L454), we should leave none to include all fields.

Reply:
Thanks for pointing that out. Updated.